exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,6 @@
-const name = " Daniel  ";
+var firstName = " Daniel  ";
+var nameTrim = firstName.trim();
+var firstNameLength = firstName.length;
```

Comment:
you should check the length of nameTrim not the name containing the extra spaces.